drivers/hmis/app/graphql/types/hmis_schema/unit.rb

```diff
@@ -65,10 +65,12 @@ def latest_opportunity
     end
 
     def accepting_ce_referrals
+      # First check for an existing opportunity. If there is none, then this unit isn't accepting referrals
       latest_opportunity = load_ar_association(object, :latest_opportunity)
-      return false if latest_opportunity.nil?
+      return false if latest_opportunity.nil? || latest_opportunity.closed?
 
-      load_ar_association(latest_opportunity, :referrals, scope: Hmis::Ce::Referral.active).empty?
+      # Otherwise, the unit is only accepting referrals if the opportunity doesn't already have an active referral.
+      load_ar_association(object, :active_referral).nil?
```

Comment:
Maybe clearer to use `active_or_accepted_referral` here, even though it should never be `accepted` because we have already filtered out `closed` opportunities?

Reply:
I added some clarifying comments, I think it's okay to leave as is.